addons/sale_stock/models/sale_order.py

```diff
@@ -270,7 +270,7 @@ class SaleOrderLine(models.Model):
     free_qty_today = fields.Float(compute='_compute_qty_at_date', digits='Product Unit of Measure')
     qty_available_today = fields.Float(compute='_compute_qty_at_date')
     warehouse_id = fields.Many2one(related='order_id.warehouse_id')
-    qty_to_deliver = fields.Float(compute='_compute_qty_to_deliver', digits='Product Unit of Measure')
+    qty_to_deliver = fields.Float(compute='_compute_qty_to_deliver', digits='Product Unit of Measure', store=True)
```

Comment:
:red_circle:

* This is a pure UI field (with its secondary field `display_qty_widget`), it's "clearly" not designed to be stored.
* Two fields are computed in `_compute_qty_to_deliver`, you should never have stored AND non stored fields computed in the same compute (inconsistent states, conflicting recomputations aka unknown danger zone)
* If you investigate `_compute_qty_to_deliver` you will see advanced code not covered by depends, which means it's strictly not expected to be stored, but computed at display time.

Reply:
Note, I say "clearly" to insist on the distinction between UI fields and stored data fields, but you'll learn to notice those red flags in the future, I don't blame you for not knowing it was bad to store that field ;).